cmd/backup-manager/app/restore/restore.go

```diff
@@ -93,11 +94,17 @@ func (ro *Options) restoreData(
 		// init pitr restore args
 		args = append(args, fmt.Sprintf("--restored-ts=%s", ro.PitrRestoredTs))
 
-		if fullBackupArgs, err := pkgutil.GenStorageArgsForFlag(restore.Spec.PitrFullBackupStorageProvider, "full-backup-storage"); err != nil {
-			return err
+		esp := v1alpha1.StorageProvider{}
+		if !reflect.DeepEqual(restore.Spec.PitrFullBackupStorageProvider, esp) {
```

Comment:
IMHO, the comparison here seems redundant? If `pkgutil.GenStorageArgsForFlag` returns error, seems that `PitrFullBackupStorageProvider` is already nil